IdentityCore/tests/MSIDAADOauth2FactoryTests.m

```diff
@@ -350,5 +381,34 @@ - (void)testAppMetadataFromResponse_whenAADTokenResponseWithoutFamilyId_shouldRe
     XCTAssertEqualObjects(metadata.familyId, @"");
 }
 
+#pragma mark - Helpers
+
+- (void)setUpEnrollmentIdsCache:(BOOL)isEmpty
+{
+    NSDictionary *emptyDict = @{};
+    
+    NSDictionary *dict = @{MSID_INTUNE_ENROLLMENT_ID_KEY: @{@"enrollment_ids": @[@{
+                                                                                     @"tid" : @"fda5d5d9-17c3-4c29-9cf9-a27c3d3f03e1",
```

Comment:
nit: just double checking if this and the rest of the id fields are meaningless :)

Reply:
Everything with the exception of homeAccountId and enrollmentId is meaningless for the purposes of most of the test files - however, I decided to keep the blob structure the way it actually is so I don't confuse anyone unfamiliar with how enrollment IDs are stored :)